src/iqm/qiskit_iqm/iqm_job.py

```diff
@@ -184,6 +184,21 @@ def status(self) -> JobStatus:
             return JobStatus.CANCELLED
         raise RuntimeError(f"Unknown run status '{result.status}'")
 
+    def queue_position(self, refresh: bool = False) -> Optional[int]:
+        """Return the position of the job in the server queue.
+
+        Note:
+            The position is not yet implemented and this function will always
+            return ``None``. The ``refresh`` argument is ignored.
+
+        Args:
+            refresh: If ``True``, re-query the server to get the latest value.
+                Otherwise return the cached value.
+
+        Returns:
+            Position in the queue or ``None`` if position is unknown or not applicable.
+        """
+
```

Comment:
I would add an explicit `return None` and a comment that this is actually not implemented and not just an accident.

Reply:
Where does it complain about the unused argument? If it is truly unused in the place where `queue_position` is called, it should either be `_ = queue_position()` or explicitly disabled with `#pylint: disable` there.